CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing
+
+Wrangler is an open source project because we believe that folks should have access, insight,
+and the opportunity to contirbute to their developer tools. Wrangler is also a product
+delivered by Cloudflare, so it's important to clarify how we think about issue triage and
+contributions.
+
+## People
+
+Wrangler is maintained by @ashleygwilliams, and her team, [Workers Developer Experience].
+
+[Workers Developer Experience]: https://github.com/orgs/cloudflare/teams/workers-devexp
+
+## Primary Issue Triage
+
+Within 3 days, any incoming issue should be triaged. Triage involves:
+
+- reading the issue and requesting any further information
+- always thank the user for submitting
```

Comment:
I doubt I (we) will do it for every issues.

Reply:
we will. it's something i would like to enforce :)